docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicyUsage.md

```diff
@@ -129,7 +137,21 @@ Accept wildcard characters: False
 ```
 
 ### -Credential
-{{Fill Credential Description}}
+Specifies the user account credentials to use to perform this task.
+The default credentials are the credentials of the currently logged on user unless the cmdlet is run from an Active Directory module for Windows PowerShell provider drive.
```

Comment:
```suggestion
The default credentials are the credentials of the user who is currently signed in unless the cmdlet is run from an Active Directory module for Windows PowerShell provider drive.
```

Reply:
"currently logged on user" is the term I have seen in technical literature so far, so I prefer to leave it that way.

Reply:
It is your completely your choice - my change was simply in accordance with the Microsoft Style Guide: "Don't use log in, login, log into, log on, logon, log onto, log off, log out, logout, or a similar term unless it appears in the UI (and you're writing instructions). Use sign in or sign out instead.

Reply:
Agreed with you, I guess the other is outdated :)